### PR TITLE
Deserialize tz

### PR DESCRIPTION
**Pull request by cleder**

### CODE REVIEW

**Comment by cleder** [comment]
fixes #102 #106 #107

**Comment by madisvain** [comment]
This pull requests seems to be very elegantly done for TZ support as it also supports current usage. @dominicrodger  There has been no review process and updated to this library since September. Things like Django 2.0 support and timezone support that has multiple tickets are critical for this library to be in a healthy state.

Time wise I certainly know how hard it's to support all the questions and pull requests. Has there been any discussion of adding core contributors to this package?

_Besides this I would like to thank Dominic for writing this library. It has helped us out of the django-celery-beat hole that we were in and helps us support recurring tasks in a much more sustainable and dynamic way. **Thank you!**_

**Comment by dominicrodger** [comment]
All valid comments @madisvain, and very kindly and thoughtfully expressed.

First, this pull request - I need to take a closer look, but I think it's likely fine to merge, I'll try to get that done by the end of next week.

Secondly, I'm very open to more maintainers - I am not actually using django-recurrence any more, and haven't done since the Django 1.8 release, so it would be great to have other contributors on board. For now, if anyone wants write access I have no major objections, and we will leave it so only @emperorcezar has permission to push to PyPI. If anyone wants to step up - let me know either on here, or by email (internet@dominicrodger.com).

Again, can't emphasize enough how much I appreciated the way you phrased your comment.

Dom

**Comment by cleder** [comment]
It would be highly appreciated if there was some documentation about the development process, who has to be prodded for code reviews, Pull requests and releases to Pypi

**Comment by dominicrodger** [comment]
Hey @cleder,

Me for code reviews/pull requests, and @emperorcezar for pushes to PyPI. As mentioned - if others want to help out with reviews/merges here, I'm happy to add people.

Dom

**Comment by dominicrodger** [comment]
Thanks for your work on this @cleder. Let me know if you're interested in being added as a contributor.

**Comment by bramski** [comment]
Please let's update the documentation. I had to search this pull request to find that this feature is hidden behind a setting change.

File: recurrence/settings.py [hunk 1]
```diff
@@ -0,0 +1,25 @@
...
+"""
+
+from django.conf import settings
+
+
+def deserialize_tz():
+    try:
+        return settings.RECURRENCE_USE_TZ
+    except AttributeError:
+        return settings.USE_TZ
```
Comment (bramski) [inline recurrence/settings.py]: This does not appear to work. My app had `USE_TZ = True` and i was running into this bug.

**Pull request merged**